owncloudApp/src/main/java/com/owncloud/android/shares/presentation/fragment/ShareFileFragment.kt

```diff
@@ -322,6 +326,17 @@ class ShareFileFragment : Fragment(), ShareUserListAdapter.ShareUserAdapterListe
         updatePublicLinkButton()
 
         // Update view depending on updated capabilities
+        if (!isShareApiEnabled) {
+            shareHeaderDivider.visibility = View.GONE
```

Comment:
why don't you use isVisible extension? This block would be refactored in:

        shareHeaderDivider.isVisible = isShareApiEnabled
        shareWithUsersSection.isVisible = isShareApiEnabled
        shareViaLinkSection.isVisible = isShareApiEnabled

Reply:
Done